lib/storage/src/content_manager/snapshots/mod.rs

```diff
@@ -44,9 +41,15 @@ pub async fn do_delete_full_snapshot(
     wait: bool,
 ) -> Result<bool, StorageError> {
     let dispatcher = dispatcher.clone();
-    let snapshot_name = snapshot_name.to_string();
-    let task =
-        tokio::spawn(async move { _do_delete_full_snapshot(&dispatcher, &snapshot_name).await });
+    let storage_config = dispatcher
+        .clone()
+        .toc()
+        .storage_config
+        .to_shared_storage_config(false);
```

Comment:
Here I am creating the shared storage config from the storage config to avoid duplicating the *snapshot_manager* method.